decoder.go

```diff
@@ -228,6 +240,7 @@ func (d *decoder) decode(nalu []byte) (*decoderOutput, error) {
 		// a fresh frame.
 		d.dst.free()
 		dstFrame, err := newAVFrameWrapper()
+		dstFrame.refs.Store(1)
```

Comment:
Use `Add(1)`. And if you're paranoid assert that "new" value is "1".

Reply:
Got it

Reply:
Done